core/src/main/java/com/amplifyframework/auth/AuthUserAttributeKey.java

```diff
@@ -46,7 +46,7 @@
      * Construct a new instance of AuthUserAttributeKey.
      * @param attributeKey the attribute key to use in the new object.
      */
-    protected AuthUserAttributeKey(String attributeKey) {
+    public AuthUserAttributeKey(String attributeKey) {
```

Comment:
You don't have to make this public. You can use `AuthUserAttributeKey.custom(CUSTOM_ATTR)`, or one of the existing `AuthUserAttributeKey.address()`, etc.